lib/mix/test/test_helper.exs

```diff
@@ -393,6 +393,7 @@ end
 defmodule Mix.Tasks.Hello do
   use Mix.Task
   @shortdoc "This is short documentation, see"
+  @requirements ["cmd touch file1.txt", "cmd touch file2.txt"]
```

Comment:
Instead of changing an existing task, define a new task in `mix/task_test.exs`. :) This particular task is used in many places. Also, it is best to not use `cmd`, since `touch` is not available on Windows. Maybe use `["help compile", "hello"]` and use `capture_io` to make sure the contents have been printed. Or similar.